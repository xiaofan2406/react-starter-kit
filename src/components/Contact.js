import React from 'react';
import { css } from '@emotion/core';
import { Loader } from 'widgets';

const Contact = () => (
  <div
    css={css`
      text-align: center;
    `}
  >
    <Loader size={120} />
  </div>
);

export default Contact;
