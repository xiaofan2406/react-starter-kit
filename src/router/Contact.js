import React from 'react';
import { Loader } from 'widgets';

function Contact() {
  return <Loader color="green" />;
}

export default Contact;
