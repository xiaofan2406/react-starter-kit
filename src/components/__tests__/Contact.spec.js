/* @flow */
import React from 'react';
import { render } from 'react-testing-library';
import Contact from '../Contact';

test('Contact component matches snapshot', () => {
  const { container } = render(<Contact />);

  expect(container.firstChild).toMatchSnapshot();
});
