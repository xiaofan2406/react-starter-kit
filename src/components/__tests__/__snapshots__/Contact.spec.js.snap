// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact component matches snapshot 1`] = `
.emotion-2 {
  text-align: center;
}

.emotion-0 {
  display: inline-block;
}

.emotion-0 > div {
  width: 120px;
  height: 120px;
  position: relative;
}

.emotion-0 > div div {
  width: 120px;
  height: 120px;
  border-style: solid;
  border-width: 12px;
  border-color: #00bcd4;
  opacity: 0.4;
  border-radius: 100%;
  position: absolute;
  top: 0;
  left: 0;
  -webkit-perspective: 800px;
  -moz-perspective: 800px;
  -ms-perspective: 800px;
  perspective: 800px;
  -webkit-animation-fill-mode: forwards;
  animation-fill-mode: forwards;
}

.emotion-0 > div div:first-child {
  -webkit-animation: loaderRigthRotate 2s 0s infinite linear;
  animation: loaderRigthRotate 2s 0s infinite linear;
}

.emotion-0 > div div:nth-child(2) {
  -webkit-animation: loaderLeftRotate 2s 0s infinite linear;
  animation: loaderLeftRotate 2s 0s infinite linear;
}

<div
  class="emotion-2"
>
  <div
    class="emotion-0 emotion-1"
    color="#00bcd4"
    size="120"
  >
    <div>
      <div />
      <div />
    </div>
  </div>
</div>
`;
